src/app/app-routing.module.ts

```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { SongListComponent } from './song/song-list/song-list.component';
+import { SongDetailsComponent } from './song/song-details/song-details.component';
+import { SongCreateComponent } from './song/song-create/song-create.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path: '', component: SongListComponent},
+  {path: 'create', component: SongCreateComponent},
+  {path: 'details/:id', component: SongDetailsComponent},
+  {path: 'edit/:id', component: SongCreateComponent},
+];
```

Comment:
Does that mean that localhost:4200/create will point me to song create component?
Why not localhost:4200/song_create or something like that?